repos/fdbt-site/tests/pages/api/searchOperators.test.ts

```diff
@@ -109,232 +110,18 @@ describe('searchOperators', () => {
-            ],
-        };
-
-        await searchOperators(req, res);
-
-        expect(updateSessionAttributeSpy).toHaveBeenCalledWith(
-            req,
-            MULTIPLE_OPERATOR_ATTRIBUTE,
-            expectedSessionAttributeCall,
-        );
-        expect(res.writeHead).toBeCalledWith(302, {
-            Location: '/searchOperators?searchOperator=warr',
-        });
-    });
-
-    it("should redirect with errors when the user clicks the 'Add Operator' button without making a selection", async () => {
-        const { req, res } = getMockRequestAndResponse({
-            requestHeaders: { referer: 'host/searchOperators?searchOperator=warr' },
-            body: {
-                addOperators: 'Add Operator(s)',
-                searchText: '',
-            },
-        });
-
-        const expectedSessionAttributeCall: MultipleOperatorsAttributeWithErrors = {
-            selectedOperators: [],
-            errors: [{ errorMessage: 'Select at least one operator to add', id: 'add-operator-checkbox-0' }],
-        };
-
-        await searchOperators(req, res);
-
-        expect(updateSessionAttributeSpy).toHaveBeenCalledWith(
-            req,
-            MULTIPLE_OPERATOR_ATTRIBUTE,
-            expectedSessionAttributeCall,
-        );
-        expect(res.writeHead).toBeCalledWith(302, {
-            Location: '/searchOperators?searchOperator=warr',
-        });
-    });
-
-    it("should succesfully remove from the selected operators when the user selects from the list and clicks the 'Remove Operator' button", async () => {
-        const { req, res } = getMockRequestAndResponse({
-            body: {
-                operatorsToRemove: ['MCTR#Manchester Community Transport', 'MCTR2#Manchester Community Transport 2'],
-                removeOperators: 'Remove Operators',
-                searchText: '',
-            },
-            session: {
-                [MULTIPLE_OPERATOR_ATTRIBUTE]: {
-                    selectedOperators: [
-                        {
-                            nocCode: 'MCTR',
-                            name: 'Manchester Community Transport',
-                        },
-                        {
-                            nocCode: 'MCTR2',
-                            name: 'Manchester Community Transport 2',
-                        },
-                        {
-                            nocCode: 'MCTR3',
-                            name: 'Manchester Community Transport 3',
-                        },
-                    ],
-                },
-            },
-        });
-
-        const expectedSessionAttributeCall: MultipleOperatorsAttribute = {
-            selectedOperators: [
-                {
-                    nocCode: 'MCTR3',
-                    name: 'Manchester Community Transport 3',
-                },
-            ],
-        };
-
-        await searchOperators(req, res);
```

Comment:
We have removed a lot of tests here but not added any new ones - we have made a lot of code changes to the functionality, do we still have all the scenarios covered?